Source/References/StateHoldr-0.2.0.ts

```diff
@@ -250,6 +260,21 @@ module StateHoldr {
         */
 
         /**
+         * Clears the list of collectionKeys.
+         */
+        clearCollectionKeys(): void {
+            this.collectionKeys = [];
+        }
+
+        /**
+         * Retrieves and sets collectionKeys to the list saved in ItemHolder.
+         */
+        retrieveCollectionKeys(): void {
+            var keys: string[] = this.ItemsHolder.getItem("collectionKeys");
+            this.collectionKeys = typeof keys === "undefined" ? [] : keys;
+        }
```

Comment:
This function should not be necessary. Why even keep `this.collectionKeys` if it's a direct copy of what's in `this.ItemsHolder`?